gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -114,6 +114,62 @@
 	return c.signAndBroadcastTxCommit(tx, cfg.AccountNumber, cfg.SequenceNumber)
 }
 
+// CallTxSync executes one or more MsgCall calls on the blockchain synchronously
+func (c *Client) CallTxSync(cfg BaseTxCfg, msgs ...MsgCall) (*ctypes.ResultBroadcastTx, error) {
```

Comment:
We should leave this for a separate PR 🙏

Reply:
will do